### PR TITLE
fix: scope local storage by specific configuration

### PR DESCRIPTION
**Pull request by joanise**
First step in fixing https://github.com/roedoejet/kawennonnis_ui/issues/6

### CODE REVIEW

**Comment by joanise** [comment]
@roedoejet I'll wait for review on this before I merge into kawe_ui and add the extra commit required there.

File: projects/word-weaver/src/app/core/local-storage/local-storage.service.ts
Comment (roedoejet) [inline projects/word-weaver/src/app/core/local-storage/local-storage.service.ts]:
```suggestion
const APP_PREFIX = "ww-" + slugify(environment.base) + slugify(CONFIG_NAME);
```

Might be worth slugifying as well? On the one hand this could unintentially create conflicts, on the other hand I'm not sure if localStorage can handle arbitrary characters

Comment (joanise) [reply projects/word-weaver/src/app/core/local-storage/local-storage.service.ts]: Yes, excellent point, we should.

**Review by roedoejet — APPROVED** [review]
Thanks @joanise !

**Pull request merged**
